test/recordlentest.c

```diff
@@ -157,11 +157,7 @@ static int test_record_overflow(int idx)
         overf_expected = 0;
     }
 
-    if (idx == TEST_ENCRYPTED_OVERFLOW_TLS1_3_OK
-            || idx == TEST_ENCRYPTED_OVERFLOW_TLS1_3_NOT_OK)
-        recversion = TLS1_VERSION;
-    else
-        recversion = TLS1_2_VERSION;
+    recversion = TLS1_2_VERSION;
```

Comment:
I'm slightly surprised that this works with no-tls12 builds, but any issue there is not new to this version, so it's clearly okay.

Reply:
It never gets this far in a no-tls12 build due to the check at the beginning of this function.